schema/test/unit/tables/project_revision_amendment_type_test.sql

```diff
@@ -82,14 +82,6 @@ select results_eq(
     'Data was changed by cif_admin'
 );
 
-select throws_like(
```

Comment:
can we replace the removed `throws_like` test with a `lives_ok` test to check that the `delete` permission works?